staging/content.pem

```diff
@@ -0,0 +1,8 @@
+-----BEGIN RSA PUBLIC KEY-----
```

Comment:
could you please name this file in the same way as the kid if this file? 
`ff24abcd-b1da-4df7-868a-b19f6f3dfb6a.pem`

Reply:
and same for prod

Reply:
Done